examples/functions-with-included-record-parameters/functions_with_included_record_parameters.description

```diff
@@ -0,0 +1,5 @@
+// A Ballerina function can have included record parameters.
+// The included record parameter must be of the `record` type and it cannot have default values.
```

Comment:
```suggestion
// The included record parameter must be of a record type and it cannot be a defaultable parameter.
```

With the current wording, it kinda feels like the record fields can't have default values.

Reply:
Fixed.